projects/demo-playwright/tests/core/error/tui-error.spec.ts

```diff
@@ -0,0 +1,19 @@
+import {tuiGoto} from '@demo-playwright/utils';
+import {expect, test} from '@playwright/test';
+
+test.describe(`TuiError`, () => {
+    test(`No extra space between content`, async ({page}) => {
+        await tuiGoto(page, `components/error#base`);
+        const example = page.locator(
+            `tui-doc-example[heading="Basic"] [automation-id="tui-doc-example"]`,
+        );
+
+        await example.locator(`[automation-id="tui-toggle__checkbox"]`).click();
```

Comment:
Let's simplify selector

```suggestion
        await example.locator(`tui-toggle input[type="checkbox"]`).click();
```

I hope there is no need to reveal the complex `tui-toggle`'s structure 🤔

Reply:
The same for `tui-error`

Reply:
```ts
const checkbox = example.locator(`tui-toggle input[type="checkbox"]`);
const error = example.locator(`tui-error`);

await expect(error).not.toBeVisible();
await checkbox.click();

await expect(checkbox).toBeChecked();
await expect(error).toBeVisible();
await expect(error).toHaveText(`An error`);

await expect(example).toHaveScreenshot(`01-error.png`);
```